Bottom: 7673305db24e0b03187acb8928ddfeca18980793
Top:    2b93df95492ad4d1ceb8aa1d1cf8582c93b425b0
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-13 22:08:59 -0300

Thread safety documentation.

for ChangeLog

	* manual/libdl.texi: New.


---

diff --git a/manual/libdl.texi b/manual/libdl.texi
new file mode 100644
index 0000000..e3fe045
--- /dev/null
+++ b/manual/libdl.texi
@@ -0,0 +1,10 @@
+@c FIXME these are undocumented:
+@c dladdr
+@c dladdr1
+@c dlclose
+@c dlerror
+@c dlinfo
+@c dlmopen
+@c dlopen
+@c dlsym
+@c dlvsym
